### PR TITLE
change to int and fix UT

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: controller/scripts/lint.ini [hunk 1]
```diff
@@ -23,6 +23,7 @@ ignore-patterns=csi_pb2.*[.]py
 # relevant categories: E - Error, W - Warning, R - Refactor, C - Convention
 disable=
  # easy:
+ raise-missing-from, # W - add "from" after some of the "raise"
```
Comment (oriyarde) [inline controller/scripts/lint.ini:26]: forgot to mention that IMO it's worth adding **`happened only on Jenkins,`**`add "from"...`

Comment (ArbelNathan) [reply controller/scripts/lint.ini:26]: as far as we know. I dont think its a fact that it cant happen in other places

Comment (oriyarde) [reply controller/scripts/lint.ini:26]: that's why it's in past tense.
should be useful for reproduction purposes

Comment (ArbelNathan) [reply controller/scripts/lint.ini:26]: done

**Review by oriyarde — APPROVED** [review]

**Review by matancarmeli7 — APPROVED** [review]

**Pull request merged**